### PR TITLE
Add spot auto scaling group 

### PR DESCRIPTION
**Pull request by jinruh**
To have a hybrid mode auto scaling

screen shot 2016-02-26 at 12 57 52 am

### CODE REVIEW

**Comment by jinruh** [comment]
@sbaogang @yujunglo

File: deploy-service/common/src/main/java/com/pinterest/arcee/bean/AutoScalingGroupType.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package com.pinterest.arcee.bean;
```
Comment (yujunglo) [inline deploy-service/common/src/main/java/com/pinterest/arcee/bean/AutoScalingGroupType.java]: Please add Copyright here.

File: deploy-service/common/src/main/java/com/pinterest/arcee/bean/SpotAutoScalingBean.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
+package com.pinterest.arcee.bean;
```
Comment (yujunglo) [inline deploy-service/common/src/main/java/com/pinterest/arcee/bean/SpotAutoScalingBean.java]: Please also add Copyright here.

File: deploy-service/common/src/main/java/com/pinterest/arcee/dao/SpotAutoScalingDAO.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
+package com.pinterest.arcee.dao;
```
Comment (yujunglo) [inline deploy-service/common/src/main/java/com/pinterest/arcee/dao/SpotAutoScalingDAO.java]: Copyright here

File: deploy-service/common/src/main/java/com/pinterest/arcee/db/DBSpotAutoScalingDAOImpl.java [hunk 4]
```diff
@@ -0,0 +1,71 @@
+package com.pinterest.arcee.db;
```
Comment (yujunglo) [inline deploy-service/common/src/main/java/com/pinterest/arcee/db/DBSpotAutoScalingDAOImpl.java]: Copyright here

**Comment by yujunglo** [comment]
Looks good to me. Thanks.

**Pull request merged**